domain/induction/src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/induction/service/InductionServiceTest.kt

```diff
@@ -44,7 +44,7 @@ class InductionServiceTest {
     fun `should create induction`() {
       // Given
       val createInductionDto = aValidCreateInductionDto()
-      val induction = aValidInduction()
+      val induction = aFullyPopulatedInduction()
```

Comment:
Strictly speaking it was never a "valid" Induction before, so this makes more sense, but it has inevitably made the diff larger.